docs/application/dotnet/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,509 @@
+
+5. To mute a media source, use the `Mute` property:
+
+    ```csharp
+    mediaSource.Mute = true;
+    ```
+    > [!NOTE]
+    > Muting a media source means it sends black video frames or silent audio frames to a remote peer. Mute or unmute of a media source is also possible in `WebRTCState.Playing` state.
+    > Some types of media sources do not support this functionality. For example, `MediaFileSource`, `MediaPacketSource`, and `MediaNullSource`.
+
+6. To set or get the encoder bitrate to a media source, use the `EncoderBitrate` property:
+
+    ```csharp
+    ...
+    await webRtc.StartAsync();
+    ...
+    var encoderBitrate = mediaSource.EncoderBitrate;
+    // some logic to increase or decrease the current target bitrate
+    // e.g. var newEncoderBitrate = encoderBitrate / 2;
+    mediaSource.EncoderBitrate = newEncoderBitrate;
+    ```
+    > [!NOTE]
+    > Some types of media sources do not support this functionality. For example, `MediaFileSource`, `MediaPacketSource`, and `MediaNullSource`.
+
+7. To set or get the video resolution to a media source, use the `VideoResolution` property:
+
+    ```csharp
+    mediaSource.VideoResolution = new Size(640, 480);
+    ```
+    > [!NOTE]
+    > Some types of media sources support dynamic resolution change while streaming. Otherwise `InvalidOperationException` exception will be thrown.
+
+8. To set or get the video frame rate to a media source, use the `VideoFrameRate()` property:
+
+    ```csharp
+    mediaSource.VideoFrameRate = 15;
+    ```
+    > [!NOTE]
+    > If the input value is not supported by the media source, the `ErrorOccurred` event will be invoked.
+
+9. To set the device id to a media source for camera, use the `CameraDeviceId` property before calling `Start()` or `StartAsync()`:
+
+    ```csharp
+    ...
+    var cameraSource = new MediaCameraSource();
+    webRtc.AddSource(cameraSource);
+    ...
+    cameraSource.CameraDeviceId = 1;
+    ...
+    await webRtc.StartAsync();
+    ```
+    > [!NOTE]
+    > If the device id is not valid, `StartAsync()` or `Start` will throw `InvalidOperationException` exception.
+
+<a name="data_channel"></a>
+## Control data channels
+
+You can create a data channel to a webrtc. It is also possible to get notified when you have a new data channel requested by a remote peer. You can send or receive data to or from these data channels by using the APIs below:
+
+1. To create a data channel, use `WebRTCDataChannel` before calling `Start()` or `StartAsync()`:
+
+    ```csharp
+    var webRtc = new WebRTC();
+    var dataChannel = new WebRTCDataChannel(webRtc, "data_channel_label");
+    ...
+    await webRtc.StartAsync();
+    ```
+
+2. To get notified when a data channel is created by a remote peer, use `WebRTC.DataChannel` event. The event will be invoked when it is created after negotiation:
+
+    ```csharp
+    webRtc.DataChannel += (s, e) => Log.Info("WebRTC", "DataChannel is created");
```

Comment:
Hello @hsgwon , I have a question regarding the variable in the code here. Since the function mentioned in Line# 196 is `WebRTC.DataChannel`, it is different from what has been used in the code. Which one is this the correct function?

Reply:
`WebRTC.DataChannel` is not event. I just want to describe DataChannel event of WebRTC class.
I'll change it to just `DataChannel` if it's confused.